src/Dialogs/Airspace/dlgAirspaceWarnings.cpp

```diff
@@ -330,6 +330,7 @@ AirspaceWarningListWidget::OnPaintItem(Canvas &canvas,
   auto [text_rc, altitude_rc] =
     text_altitude_rc.VerticalSplit(text_altitude_rc.right - (padding + altitude_width));
   text_rc.right -= padding;
+  altitude_rc.bottom = altitude_rc.bottom;  // unused variable?
```

Comment:
Doesn't belong in this commit

Reply:
Ok, I would move it in an extra commit. Is it possible to do this in this PR?

Reply:
Yes.